src/stack.js

```diff
@@ -1,5 +1,19 @@
 class Stack {
+  constructor() {
+    this.stack = [];
+  }
 
+  add(value) {
+    this.stack.push(value);
+  }
+  remove() {
+    const valueRemoved = this.stack.pop();
```

Comment:
You can simply do `return this.stack.pop();` without having to store a reference to the return value of `pop()`.